aes-gcm/tests/xaes256gcm.rs

```diff
@@ -0,0 +1,35 @@
+//! XAES-256-GCM test vectors
+
+#![cfg(all(feature = "aes", feature = "alloc"))]
+
+#[macro_use]
+mod common;
+
+use aes_gcm::aead::{array::Array, Aead, AeadInPlace, KeyInit, Payload};
+use aes_gcm::XaesGcm256;
+use common::TestVector;
+use hex_literal::hex;
+
+/// C2SP XAES-256-GCM test vectors
```

Comment:
sidebar - would love if C2SP test vectors would be embedded in crate a-like of [wycheproof-rs](https://crates.io/crates/wycheproof) so test vectors can be updated independently to all associated impl's and no messing with blobs / updating multiple places